packages/ckeditor5-package-generator/lib/templates/js-legacy/package.json

```diff
@@ -24,7 +24,7 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">=20.0.0",
+    "node": ">=22.0.0",
     "npm": ">=5.7.1"
```

Comment:
Let's remove these properties from all files (**a generated package only**). If somebody used the tool, its environment matches the requirements already:

https://github.com/ckeditor/ckeditor5-package-generator/blob/aac6709fa28ba12e8b8c1a327de6025e61ff1a81/package.json#L41-L43

The generated package does not use any Node.js feature that requires a specific version. It's enough that a package works with any active LTS version.